src/web/app/pages-instructor/instructor-course-edit-page/instructor-course-edit-page.component.html

```diff
@@ -1,3 +1,368 @@
+              <i class="fa fa-trash" aria-hidden="true"></i>
+              Delete
+            </button>
+          </span>
+        </div>
+        <div class="card-body">
+          <form class="form form-horizontal" [formGroupName]="i" (ngSubmit)="onSubmitEditInstructor(instr, i)">
+            <div class="form-row form-group">
+              <label class="col-sm-3 control-label">
+                Google ID:
+              </label>
+              <div class="col-sm-9" *ngIf="hasGoogleId(i)">
+                <input class="form-control immutable" maxlength="254" tabindex="3" formControlName="googleId">
+              </div>
+              <span class="text-warning" *ngIf="!hasGoogleId(i)">
+                Not available. Instructor is yet to join this course.
+              </span>
+            </div>
+            <div class="form-row form-group">
+              <label class="col-sm-3 control-label">
+                Name:
+              </label>
+              <div class="col-sm-9">
+                <input class="form-control" ngbTooltip="Enter the name of the instructor." maxlength="100" tabindex="4" formControlName="name">
+              </div>
+            </div>
+            <div class="form-row form-group">
+              <label class="col-sm-3 control-label">
+                Email:
+              </label>
+              <div class="col-sm-9">
+                <input class="form-control" ngbTooltip="Enter the email of the instructor." maxlength="254" tabindex="5" formControlName="email">
+              </div>
+            </div>
+            <div class="form-row form-group">
+              <label class="col-sm-3 control-label">
+                <input checked="" ngbTooltip="If this is unselected, the instructor will be completely invisible to students. E.g. to give access to a colleague for ‘auditing’ your course"
+                       type="checkbox" formControlName="isDisplayedToStudents" (change)="onChangeIsDisplayedToStudents($event, instr, i)">
+                Display to students as:
+              </label>
+              <div class="col-sm-9">
+                <input class="form-control" id="name-display-{{ i }}" ngbTooltip="Specify the role of this instructor in this course as shown to the students"
+                       type="text" formControlName="displayedName" [placeholder]="getPlaceholderForDisplayedName(instr.controls.isDisplayedToStudents.value)">
+              </div>
+            </div>
+            <div>
+              <div class="form-row form-group">
+                <label class="col-sm-3 control-label">
+                  Access Level:
+                </label>
+                <div class="col-sm-9">
+                  <span class="form-control-static" id="role-view-{{ i }}">
+                    {{ instructorList[i].role }}
+                    <a class="view-role-details" href="#" (click)="viewInstructorRole(viewInstructorRoleModal, i)">
```

Comment:
`href="#"` will bring you to the top of the page. You're likely looking for `href="javascript:;"`?